client/stylesheets/_toCat.scss

```diff
@@ -4813,13 +4814,14 @@ On click the images will be displayed at normal size to complete the effect
 /*Options DropDown*/
 
 .optionsClick {
-	margin:100px auto;
-	width:200px;
+    width: 100%;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered margin-bottom, margin-top, width